### DISCUSSION TITLE
Add AUD (Australian Dollar) as fiat currency

### DISCUSSION THREAD

**Discussion by nit27** [alephium/alephium-frontend#421] [Feature requests]
G'day mate 🤙

Since I found many fiat currencies on DW & MW, but there is still no AUD (Australian Dollar). Therefore, I wish you could add AUD as fiat display for spreading the awesome alephium project to Aussies 🇦🇺

Many thanks. Cheers!

**Comment by parispham** [comment]
Terrific idea! good on ya!

**Comment by nop33** [comment]
This is already available in the desktop wallet. It's coming up in the next mobile wallet version as well! Closing.

**Reply by nit27** [reply]
we've just seen. That's amasing! Thank you so much!
